solr/solr-ref-guide/modules/getting-started/pages/tutorial-films.adoc

```diff
@@ -340,13 +352,13 @@ If we construct a query that looks like this:
 
 [source,bash]
 curl 'http://localhost:8983/solr/films/select?q=*:*&rows=0'\
-    '&facet=true'\
-    '&facet.range=initial_release_date'\
-    '&facet.range.start=NOW-20YEAR'\
-    '&facet.range.end=NOW'\
-    '&facet.range.gap=%2B1YEAR'
+'&facet=true'\
```

Comment:
Why no indent?

Reply:
curl doesn't handle the whitespace indent when copy/pasting it, treats each set of `'&facet...` as a separate whitespace delimited argument. The `\` escapes the newline, but doesn't help with melding into a single arg.

Reply:
Ah, did not know that.